charts/internal/seed-controlplane/charts/csi-driver-controller/templates/deployment-csi-driver-controller.yaml

```diff
@@ -131,7 +131,10 @@ spec:
         - --leader-election
         - --leader-election-namespace=kube-system
         - --timeout={{ .Values.timeout }}
-        - --v=5
+        - --retry-interval-start={{ .Values.retryIntervalStart }}
```

Comment:
Maybe it is an idea to check if the value is set before, to be able to use the upstream default if no value is set. Than also the defaults in the values.yaml can be commented.

Reply:
I don't like values in values.yml beeing commented. It's pointless, why not have them defined.